developer/src/kmldmlc/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@keymanapp/ldml-keyboard-compiler",
+  "description": "Keyman Developer LDML keyboard compiler",
+  "keywords": [
+    "keyboard",
+    "keyman",
+    "ldml",
+    "unicode"
+  ],
+  "scripts": {
+    "build": "tsc -b",
+    "test": "cd src/test && tsc -b && cd .. && mocha",
+    "prepublishOnly": "npm run build"
+  },
+  "author": "Marc Durdin <marc@keyman.com> (https://github.com/mcdurdin)",
+  "license": "MIT",
+  "main": "build/kmldmlc.js",
+  "bin": {
+    "kmldmlc": "build/kmldmlc.js"
+  },
+  "dependencies": {
+    "commander": "^3.0.0",
+    "typescript": "^4.5.4",
+    "xml2js": "^0.4.19"
+  },
+  "devDependencies": {
+    "@keymanapp/keyman-version": "*",
+    "@types/chai": "^4.1.7",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.14.6",
+    "@types/xml2js": "^0.4.5",
+    "chai": "^4.3.4",
+    "chalk": "^2.4.2",
+    "mocha": "^8.4.0",
+    "ts-node": "^9.1.1"
+  },
+  "mocha": {
+    "spec": "build-tests/**/test-*.js"
+  }
```

Comment:
```suggestion
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/keymanapp/keyman.git"
  }
```

Reply:
Yeah, I'd left the repo out because it was still kinda internal... but I guess it doesn't hurt!